.travis.yml

```diff
@@ -38,6 +41,12 @@ jobs:
         - wget https://scrutinizer-ci.com/ocular.phar
         - php ocular.phar code-coverage:upload --format=php-clover clover.xml
 
+    - stage: Test
+      php: nightly
+      before_script:
+        - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{.disabled,}
```

Comment:
This is already done in `before_install` (in a weird way, BTW, the documented way is `- phpenv config-rm xdebug.ini`), see https://docs.travis-ci.com/user/languages/php/

Reply:
Ah, you're right. This should actually override `before_install`, I just used the wrong key. I've copied the original line and would suggest changing this in a future release.

Reply:
Yes let's not confuse people.